src/utils.c

```diff
@@ -67,3 +71,95 @@ void rv_clock_gettime(struct timespec *tp)
     tp->tv_sec = tv_sec;
     tp->tv_nsec = tv_usec / 1000; /* Transfer to microseconds */
 }
+
+char *sanitize_path(const char *input)
+{
+    size_t n = strnlen(input, MAX_PATH_LEN);
+
+    char *ret = malloc(n + 1);
+    memset(ret, '\0', n + 1);
```

Comment:
Use `calloc` instead of `malloc` + `memset`.
See https://vorpus.org/blog/why-does-calloc-exist/

Reply:
Thanks for the reference